SecretSanta/src/SecretSanta.Business/Gift.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SecretSanta.Business
+{
+    public class Gift
+    {
+        public int Id { get; private set; }
```

Comment:
This is not a readonly property. To make it readonly, simply drop the setter.
```suggestion
        public int Id { get; }
```